payments/invoice/other-features.md

```diff
@@ -177,6 +177,46 @@ Content-Type: application/json
 }
 ```
 
+Customers should be able to set `recurrenceToken` to deleted if the consumer 
+deletes their agreements with their customer. 
```

Comment:
We need to explain how the operation that describes how to delete a token is found. What is its `rel` name? Please provide an example and update the `<OPERATION NAME>` in the description.

```suggestion
Payers should be able to delete payment tokens that are associated to
them. To delete a `recurrenceToken`, perform the `<OPERATION NAME>`
operation.

```